### PR TITLE
Fixed missing v6 enable flag in macvlan doc

### PR DESCRIPTION
**Pull request by nerdalert**
Signed-off-by: Brent Salisbury <brent@docker.com>

### CODE REVIEW

File: docs/macvlan.md [hunk 1]
```diff
@@ -232,12 +232,16 @@ Hosts on the same VLAN are typically on the same subnet and almost always are gr
 
 The following specifies both v4 and v6 addresses. An address from each family will be assigned to each container. You can specify either family type explicitly or allow the Libnetwork IPAM to assign them from the subnet pool.
 
+*Note on IPv6:* When declaring a v6 subnet with a `docker network create`, the flag `--ipv6` is required along with the subnet (in the following example `--subnet=2001:db8:abc8::/64`). Similar to IPv4 functionality, if a IPv6 `--gateway` is not specified, the first usable address in the v6 subnet is inferred and assigned as the gateway for the broadcast domain.
+
+The following example creates a network with multiple IPv4 and IPv6 subnets to a network. The network is attached to a sub-interface of `eth0.218`. By specifying `eth0.218` as the parent, the driver will create the sub-interface (if it does not already exist) and tag all traffic for containers in the network with a VLAN ID of 218. The physical switch port on the ToR (top of rack) network port needs to have 802.1Q trunking enabled for communications in and out of the host to work.
```
Comment (aboch) [inline docs/macvlan.md]: `The following example creates a network with multiple IPv4 and IPv6 subnets to a network.`

Comment (nerdalert) [inline docs/macvlan.md]: Thanks @aboch!

File: docs/macvlan.md [hunk 2]
```diff
@@ -285,7 +299,64 @@ $ ip -6 route
   2001:db8:abc4::/64 dev eth0  proto kernel  metric 256
   2001:db8:abc8::/64 dev eth0  proto kernel  metric 256
   default via 2001:db8:abc8::10 dev eth0  metric 1024
+  
+# Containers can have both v4 and v6 addresses assigned on a single container
```
Comment (aboch) [inline docs/macvlan.md]: Can we rephrase to
`Containers can have both v4 and v6 addresses assigned to their interfaces` or
`Both v4 and v6 addresses can be assigned to the container's interface`

Comment (nerdalert) [inline docs/macvlan.md]: Done! tyvm @aboch

**Comment by aboch** [comment]
LGTM

**Pull request merged**